stories/module-analytics-settings.stories.js

```diff
@@ -76,6 +76,7 @@ storiesOf( 'Analytics Module/Settings', module )
 		decorators: [
 			withRegistry,
 		],
+		padding: 0,
```

Comment:
Same with the settings. We need to use the default padding for all settings stories in all modules.